src/prowjobsscraper/equinix_usages.py

```diff
@@ -108,4 +108,68 @@ def get_project_usages(
             headers={self._EQUINIX_ENDPOINT_HEADER: self._project_token},
         ).json()["usages"]
         logger.info("%s usages retrieved successfully", len(equinix_project_usages))
-        return [EquinixUsage.parse_obj(usage) for usage in equinix_project_usages]
+        return self._process_usages(
```

Comment:
Be careful with generic terms like "process". If we can find better describing term, it will make this code so much easier to read and not forcing the reader to dive deeper into functions.